pages/mdtutorial/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Home - When Flag Clicked</title>
+    <meta property="og:title" content="When Flag Clicked" />
```

Comment:
Fix this

Reply:
the space at the end?

Reply:
No, fix it so the title it not "Home" and also change the og:title too